src/main/java/frc/robot/subsystems/ElevatorIOVortex.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkFlex;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.config.SparkFlexConfig;
+import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
+
+public class ElevatorIOVortex implements ElevatorIO {
+    
+    private final SparkFlex elevatorMotorLeader = new SparkFlex(ElevatorConstants.leaderCAN, MotorType.kBrushless);
+    private final SparkFlex elevatorMotorFollower = new SparkFlex(ElevatorConstants.followerCAN, MotorType.kBrushless);
+
+    public ElevatorIOVortex () {
+
+        SparkFlexConfig config = new SparkFlexConfig();
+
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(ElevatorConstants.currentLimit).voltageCompensation(12.0);
+
+        elevatorMotorLeader.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        elevatorMotorFollower.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
```

Comment:
Within the configurations, we also need a setting for if the motor is reversed. You can include a `reversed` setting in the constants file as a boolean value. Then, you should configure the motors using `.setReversed(true/false)`.